generator/sequence_transform_data.py

```diff
@@ -26,21 +26,19 @@
   widht         -> width
 For full documentation, see QMK Docs
 """
-
+import os.path
 import sys
 import textwrap
 import json
 from typing import Any, Dict, Iterator, List, Tuple
-from string import ascii_letters, digits
+from string import digits
+from pathlib import Path
 
-from milc import cli
 
-from qmk.commands import dump_lines
-from qmk.constants import GPL2_HEADER_C_LIKE, GENERATED_HEADER_C_LIKE
-from qmk.keyboard import keyboard_completer, keyboard_folder
-from qmk.keymap import keymap_completer, locate_keymap
-from qmk.path import normpath
+THIS_FOLDER = Path(__file__).parent
 
+OUT_FILE = THIS_FOLDER / "magic_data.h"
```

Comment:
sequence_transform.c expects this file to be named "sequence_transform_data.h"
```suggestion
OUT_FILE = THIS_FOLDER / "sequence_transform_data.h"
```